hardware_enablement/release-notes/kmm-release-notes.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="kmm-release-notes-v2-2"]
+= Kernel Module Management Operator release notes
+include::_attributes/common-attributes.adoc[]
+:context: kmm-release-notes-v2-2
+
+toc::[]
+
+[id="kmm-2-2"]
+== Release notes for Kernel Module Management Operator 2.2
+
+=== New features
+
+// TELCODOCS-2022
+* KMM is now using the CRI-O container engine to pull container images in the worker pod instead of using HTTP calls directly from the worker container. For more information, see xref:../kmm-kernel-module-management.adoc#kmm-example-cr_kernel-module-management-operator[Example Module CR].
```

Comment:
I would probably start with:

```suggestion
* Kernel Module Management (KMM) now uses  the CRI-O container engine to pull container images in the worker pod instead of using HTTP calls directly from the worker container. For more information, see xref:../kmm-kernel-module-management.adoc#kmm-example-cr_kernel-module-management-operator[Example Module CR].
```